docker_test.go

```diff
@@ -83,3 +83,23 @@ func TestPortMappingLabels(t *testing.T) {
 	assert.NotNil(t, pm.Labels)
 	assert.Equal(t, 0, len(*pm.Labels))
 }
+
+func TestExternalVolumes(t *testing.T) {
+	container := NewDockerApplication().Container
+
+	container.Volume("", "cp", "RW")
+	ev := (*container.Volumes)[0].ExternalVolume("myVolume", "dvdi")
+
+	ev.AddOption("prop", "pval")
+	ev.AddOption("dvdi", "rexray")
+
+	assert.Equal(t, (*container.Volumes)[0].ContainerPath, "cp")
```

Comment:
The test should not need to care about fields unrelated to `ExternalVolume`, so I'd drop the check on `ContainerPath`.